src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -108,7 +111,11 @@ def _get_docker_status_and_version(ignore_errors, yes):
         output, warning, stderr = _subprocess_communicate([docker_command, "pull", IMAGE])
 
         if stderr:
-            _handle_error(DOCKER_PULL_ERROR.append_error_message(stderr), ignore_errors)
+            if DOCKER_PULL_WRONG_PLATFORM in stderr:
+                print_pass("Docker pull of '{}'".format(IMAGE))
```

Comment:
in cli we don't use print but logger, just checked this print_pass is used several places in this file. what scenario it's used for besides normal logger methods, could you pls change to logger.xxx?

Reply:
For `print_pass`, it's redirected to std_err. I think it should be fine.
https://github.com/Azure/azure-cli/blob/f5cbbb28b388ad1dd114ffaedea26446fe5acc8c/src/azure-cli/azure/cli/command_modules/acr/check_health.py#L37

Other information like versions is also printed in stderr, how do users use the result of `acr check-health` command in a script?

Reply:
Thanks, I've replaced print() with logger.warning().

> Other information like versions is also printed in stderr, how do users use the result of `acr check-health` command in a script?

`az acr check-health` is for troubleshooting purpose, usually users would not need the output of this command in script. A exit code is enough to check the status.